_blogposts/2020-09-23-release-8-3.md.raw

```diff
@@ -0,0 +1,119 @@
+---
+author: hongbo
+date: "2020-09-23" 
+previewImg:
+category: compiler
+title: What's new in ReScript 8.3 (Part 1)
+description: |
+---
+
+
+# What's new in ReScript 8.3 (Part 1)
+
+ReScript is a soundly typed language with an optimizing compiler focused on JS platform. 
+It's focused on type safety, performance and JS interop. It used to be called BuckleScript.
+
+ReScript@8.3 is available, people can try it via
+
+```
+npm i bs-platform@8.3.0
+```
+
+The changes are listed [here](https://github.com/rescript-lang/rescript-compiler/blob/master/Changes.md#83), this is a large release,  and we will go through some highlighted changes.
+
+
+
+## Lightweight FFI attributes without `bs.` prefix
+
+In this release, we make it optional to have `bs.` prefix it or not, this will make the FFI less verbose,
```

Comment:
In this release, we make the `bs.` prefix optional. This will make the FFI less verbose.